forums/migrations/0001_initial.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2019-07-17 03:48
+from __future__ import unicode_literals
```

Comment:
Please delete this migration file.
1. forums/migrations/0001_initial.py 

manage.py will ask you to create this forum migration . Try ignoring it.

Reply:
ok